### PR TITLE
Add initial sample blueprints

### PR DESCRIPTION
**Pull request by markebalch**
Two sample blueprints for BC.  LMK if any issues.

### CODE REVIEW

File: quick-start/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Blueprints quick-start
```
Comment (morgante) [inline quick-start/README.md:1]: I'm not sure we should nest these under a "quick-start" section.

File: quick-start/bucket/Kptfile [hunk 2]
```diff
@@ -0,0 +1,46 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: bucket-demo
```
Comment (morgante) [inline quick-start/bucket/Kptfile:4]: Blueprint name should match directory name (`bucket`) and generally we should avoid "demo" blueprints.

File: quick-start/bucket/Kptfile [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+openAPI:
+  definitions:
+
```
Comment (morgante) [inline quick-start/bucket/Kptfile:7]: Please remove superfluous whitespace between lines.

File: quick-start/bucket/Kptfile [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    io.k8s.cli.setters.name:
+      x-k8s-cli:
```
Comment (morgante) [inline quick-start/bucket/Kptfile:9]: This should be a required setter.

File: quick-start/bucket/Kptfile [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+          name: bucket-name
+          pattern: ${PROJECT_ID}-${NAME}
+          values:
+          - marker: ${PROJECT_ID}
+            ref: '#/definitions/io.k8s.cli.setters.project-id'
+          - marker: ${NAME}
+            ref: '#/definitions/io.k8s.cli.setters.name'
+
+    io.k8s.cli.setters.project-id:
+      x-k8s-cli:
```
Comment (morgante) [inline quick-start/bucket/Kptfile:34]: This should be a required setter.

File: quick-start/data-demo/Kptfile [hunk 6]
```diff
@@ -0,0 +1,70 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: data-demo
```
Comment (morgante) [inline quick-start/data-demo/Kptfile:4]: How does data-demo differ from bucket demo / what's the purpose of this blueprint? We should have a README for all blueprints.

**Review by morgante — CHANGES_REQUESTED** [review]

**Comment by markebalch** [comment]
Changes made

**Review by morgante — APPROVED** [review]

**Pull request merged**